### PR TITLE
Update to wdlTools 0.8.4

### PR DESCRIPTION
**Pull request by jdidion**
This fixes issues compiling some complex workflows that were rendered incorrectly by the code generator.

Note: I have been unable to reduce the issues with the customer workflow to a simple reproducible test case. I have tested locally that the customer's workflow compiles but I do not want to add the customer's workflow as a test case.

### CODE REVIEW

File: src/test/scala/dx/compiler/CompilerTest.scala [hunk 1]
```diff
@@ -957,6 +957,27 @@ class CompilerTest extends AnyFlatSpec with Matchers with BeforeAndAfterAll {
...
+  }
+
+  it should "compile workflow with call to task having optional output" taggedAs NativeTest in {
+    val path = pathFromBasename("compiler", basename = "optional_call_output.wdl")
+    val args = path.toString :: cFlags
+    val retval = Main.compile(args.toVector)
+    retval shouldBe a[Success]
+  }
+
+  it should "compile myriad workflow" taggedAs NativeTest in {
```
Comment (commandlinegirl) [inline src/test/scala/dx/compiler/CompilerTest.scala:974]: It looks like this was left here by mistake.

**Review by commandlinegirl — CHANGES_REQUESTED** [review]

File: src/test/scala/dx/compiler/CompilerTest.scala (hunk 1 above)
Comment (jdidion) [reply src/test/scala/dx/compiler/CompilerTest.scala]: oops - removed it

**Review by commandlinegirl — APPROVED** [review]
LGTM, thanks!

**Review by commandlinegirl — CHANGES_REQUESTED** [review]
Please when you get a chance check the integration tests as described here: https://jira.internal.dnanexus.com/browse/APPS-245

**Comment by jdidion** [comment]
Abandoning this in favor of #7.

**Pull request closed without merging**